view/frontend/templates/main/enhanced_pixel.phtml

```diff
@@ -1,6 +1,23 @@
 <script type="text/javascript">
     adroll_conversion_value_in_dollars = <?php echo $block->getConversionValueInDollars() ?>;
     adroll_custom_data = {"ORDER_ID": "<?php echo $block->getIncrementId() ?>"};
+
+    function getTransCookie() {
+      if (document.cookie.match('(^|;) ?__xlid=([^;]*)(;|$)')) {

```

Comment:
What is this regex intended to match?

Reply:
Document.cookie it will return a list of all domain cookies separated by semi-colon I'm looking for a key/value pair for "__xlid" within the long string. It is intended to match any occurrence of '__xlid=$1;'